eng/scripts/Language-Settings.ps1

```diff
@@ -6,30 +6,24 @@ $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
 
-function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
-{
+function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName) {
```

Comment:
Please undo these formatting changes that move the brace.

Reply:
My vs code insist helping me format JS code. :(